x-pack/plugins/cloud_security_posture/public/pages/rules/rules.test.tsx

```diff
@@ -63,6 +66,7 @@ describe('<Rules />', () => {
     const Component = getTestComponent({ packagePolicyId: '1', policyId: '2' });
     const request = {
       status: 'error',
+      isError: true,
```

Comment:
seems weird to pass both `status: error` and `isError` 

we can use just the `status`, as that's the `tag` in this tagged union

Reply:
I don't think it's weird, if anything I think we should use the `getReactQueryFixture` here so we don't have to write those requests on our own. yeah, it makes us move devoted to react-query but I think that's a good thing and we shouldn't fight it just for the slim chance that we will drop it. we can benefit from it and we already have some tools around it that works well so I think we should embrace them

I didn't change the requests in those tests to use the `reactQueryFixtures` because I didn't want to touch stuff that are out of bound of this PR. so only stuff that are directly involved with the `CspPageTemplate` were changed.